src/device-registry/models/Event.js

```
@@ -1610,11 +1602,19 @@
   return average;
 }
 
+function filterNull(data) {
+  if (!Array.isArray(data)) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'filterNull'. Consider to delete it or rename it.

'filterNull' is defined here:


(lint/suspicious/noRedeclare)